ta/pkcs11/scripts/dump_ec_curve_params.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Why do we need this script with TA ?

Reply:
When I was originally looking at `processing_ec.c`'s DER byte arrays I was wondering are they correct. There were some commands there in comments that gave data but not in easily comparable way. Then I started wondering can it be done in easier way that I can just generate curve details and then compare that they are correct.

So the script is there to enable easier generation and verification of curve details. My idea is that if new curves are added then one adds in this script just the curve name and then generates new output and merges that into `processing_ec.c`. Then one doing the review can easily just execute the script themselves and verify that they are actually correct.

Reply:
It's not ideal but it's the best we currently have, so i'm fine with this helper script.
The issue with DER parsing is that there can be several ways to encode data. Using prebuilt DER sequences looks safer to me than implementing a safe generic DER parser.